contrib/python/test/test_containers.py

```diff
@@ -21,42 +22,38 @@ def setUp(self):
         self.host = os.environ['PODMAN_HOST']
 
         self.pclient = podman.Client(self.host)
-        self.ctns = self.loadCache()
-        # TODO: Change to start() when Implemented
-        self.alpine_ctnr.restart()
+        self.loadCache()
 
     def tearDown(self):
         pass
 
     def loadCache(self):
-        with podman.Client(self.host) as pclient:
-            self.ctns = list(pclient.containers.list())
+        self.ctns = list(self.pclient.containers.list())
```

Comment:
Nit:  I'm not real sure what's inside of 'ctns' when I read just the variable.  A more descriptive name would be nice.   Especially since 'ctnr' is used in man places and on a quick read it makes you think for a second.

Reply:
I don't care as much in the testing code.  But can make that containers now that it's used throughout the file.